src/test/java/org/broadinstitute/hellbender/tools/SplitNCigarReadsIntegrationTest.java

```diff
@@ -53,4 +53,15 @@ public void testSplitsFixNDN() throws Exception {
                 Arrays.asList(getTestDataDir() +"/" + "expected.splitNCigarReadsSnippet.splitNcigarReads.fixNDN.bam"));
         spec.executeTest("test fix NDN", this);
     }
+
+    /**
+     * Currently disabled due to issue: https://github.com/broadinstitute/gatk/issues/1865
+     */
+    @Test(enabled = false) //regression test for https://github.com/broadinstitute/gatk/pull/1853
```

Comment:
Have you tested whether setting `presorted = false` in the writer fixes this? This is what GATK3 does, as far as I can tell, to address the sorting issue (this forces the writer to sort the reads for you before writing them out). If that change does fix this test, recommend making it here in this branch rather than introducing a disabled test.